mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the multitaper_psd
+    function found in the time_frequency module. Running the transform
+    method will create attributes for the estimated psd as well as the
+    frequencies used, on top of return these values.
```

Comment:
It means that `freqs` and `data` will be created as attributes when you call `transform`, on top of being returned by `transform`. (though it's only data that is returned, not freqs...it's a poorly written phrase).